content/posts/pull-from-upstream.md

```diff
@@ -0,0 +1,116 @@
+(`rawhide`) of the dist-git repository in the Packit configuration file (see
+[our documentation](/docs/configuration#pull_from_upstream)). The `upstream_project_url` needs to be a URL
+pointing to a Git repository so that we can do `git` commands on it.
+
+## `pull_from_upstream` in action
+
+Let's showcase the new job in action for the latest release of
+[Packit itself](https://pypi.org/project/packitos/).
+
+As you can see in the `Setup` section, the Upstream Release Monitoring is configured:
+there is a PyPI project `packitos` in Anitya
+with configured mapping to the Fedora package `packit` and the monitoring in the
+[`packit` dist-git repository](https://src.fedoraproject.org/rpms/packit) is enabled.
+We could configure the mapping in Anitya from the GitHub project directly instead, and it would work as well. Just be aware that
+for each Fedora package, there can be a mapping only from one project.
+
+In Packit configuration file, we have configured the job and related options:
+
+```yaml
+upstream_project_url: https://github.com/packit/packit
+issue_repository: https://github.com/packit/packit
+copy_upstream_release_description: true
+
+jobs:
+  - job: pull_from_upstream
+    trigger: release
+    dist_git_branches:
+      - fedora-all
+      - epel-8
+```
+
+You can see that version `0.66.0` of Packit in PyPI was released:
+![Packit release PyPI](/images/packit-release-pypi.png)
+
+When Upstream Release Monitoring retrieved this new version, it created a new bug:
+![Bugzilla](/images/packit-release-bugzilla.png)
+
+This triggered Packit, and after checking the Packit configuration in dist-git
+and finding the `pull_from_upstream` job, this job was run.
+
+Using the `upstream_project_url` from the configuration, Packit was able to get the needed information
+from the corresponding GitHub release:
+![Packit release GitHub](/images/packit-gh-release.png)
+
+As a result, pull requests for configured branches were created.
+Here is an example of one of the created pull requests and part of its content:
+![Dist-git PR](/images/pull-from-upstream-pr.png)
+![Pull request content](/images/pull-from-upstream-content.png)
+
+Since we have configured the [`issue_repository`](/docs/configuration#issue_repository), we could be
+also notified about errors:
+![Pull from upstream issue](/images/pull-from-upstream-issue.png)
+Currently, retriggering is not possible, but it's in our
+[plan](https://github.com/packit/packit-service/issues/1829) to implement it soon.
+
+## Few words in the end
+
+`pull_from_upstream` has just been implemented; therefore, we encourage you to help
+test it out and make it perfect! We believe this functionality
+could be beneficial for maintainers of Fedora packages and could even be integrated further.
+Any [suggestions](https://github.com/packit/packit-service/issues/new) and feedback are welcomed
+(see [contacts](https://packit.dev/#contact)).
```

Comment:
Nice!

You can also mention follow-up downstream jobs and/or https://packit.dev/docs/fedora-releases-guide.
(For some users, the `pull_from_upstream` job might be the reason to use Packit and their first job.)